### ISSUE TITLE
Mangohud doesn't need separate options for OpenGL/Vulkan

### ISSUE THREAD

**Issue by wyndon** [lutris/lutris#4126]
Currently we have 3 choices to enable Mangohud :
- Vulkan
- OpenGL
- OpenGL (32 bits)

But there's no need to have separate options for Vulkan and OpenGL (64). Because setting `MANGOHUD_DLSYM=1` (required for OpenGL) doesn't affect anything when using Vulkan. In fact, even setting `MANGOHUD=1` isn't needed because we're already prefixing the command to launch the game with the `mangohud` command.

To test what I'm talking about, disable Mangohud globally and manually add the environment variable and the command prefix in the settings (Preferences -> Global options). Then, try a game that's using Vulkan, then one with OpenGL, and you'll see that Mangohud does work even on Vulkan.

![image](https://user-images.githubusercontent.com/72203260/156319997-29b31510-d8de-44de-9f07-985c00d8e26f.png)

To sum it up, we simply only need 2 options :
- 64 bits
- 32 bits

and both of them would set `MANGOHUD_DLSYM=1`. The only difference between them would be the command's prefix.

**Comment by strycore** [comment]
Thanks, that's good to know! I'll simplify the option
